openvino_xai/explainer/explanation.py

```diff
@@ -36,8 +37,9 @@ def __init__(
         self,
         saliency_map: np.ndarray | Dict[int | str, np.ndarray],
         targets: np.ndarray | List[int | str] | int | str,
+        task: Task,
         label_names: List[str] | None = None,
-        metadata: Dict[Task, Any] | None = None,
+        predictions: Dict[Task, Prediction] | None = None,
```

Comment:
```suggestion
        predictions: Dict[int, Prediction] | None = None,
```
If I understand correctly.